### DISCUSSION TITLE
Welcome to Facebook.ApiClient Discussions!

### DISCUSSION THREAD

**Discussion by ketanjawahire** [ketanjawahire/FacebookClient#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ReneBrandon** [comment]
Any chance this will be updated to the latest Facebook API?
